src/_sass/_banner.scss

```diff
@@ -0,0 +1,11 @@
+@use "./colours";
+@use "./typography";
+
+.phase-banner {
+    background-color: colours.$yellow-light;
+    font-family: typography.$heading-font-family;
+    font-weight: typography.$heading-font-weight;
+    font-size: 1rem;
+    display: flex;
+    padding: 0 1rem;
+}
```

Comment:
Might be good to add `margin: 0.67em 0;` to match the `h1` - the spacing between the banner and the blue bar at the top of the page looks a little tight to me